README.md

```diff
@@ -17,17 +17,23 @@
    The Design System that powers Razorpay
 <p align="center">
 
-## 📦 Packages
-Blade has 2 packages
-
 ### [`@razorpay/blade`](https://github.com/razorpay/blade/tree/master/packages/blade)
-This package is under active development. Documentation for this package can be found [here](https://master--61c19ee8d3d282003ac1d81c.chromatic.com)
-- **Contributing guidelines** can be found [here](./CONTRIBUTING.md)
 
-### [`@razorpay/blade-old`](https://github.com/razorpay/blade-old)
-This package is under **maintenance** and it won't have any major releases. It will be **deprecated** once the newer version (`@razorpay/blade`) is ready for a stable release. 
-- **Documentation** for this package can be found [here](https://github.com/razorpay/blade/blob/master/packages/blade-old/README.md)
-- **Storybook** for this package can be found [here](https://blade-old.vercel.app)
+This package is under active development.
+
+- [Documentation](https://master--61c19ee8d3d282003ac1d81c.chromatic.com)
+- [Installation Guide](https://master--61c19ee8d3d282003ac1d81c.chromatic.com/?path=/docs/guides-installation--page)
+- [Contributing guidelines](./CONTRIBUTING.md)
+
+
+------
+
+> **Note**
+>
+> The blade-old package has been moved to it's own repo.
+> [github.com/razorpay/blade-old](https://github.com/razorpay/blade-old)
+> 
+> It is under **maintenance** and it won't have any major releases. It will be **deprecated** once the newer version (`@razorpay/blade`) is ready for a stable release. 
```

Comment:
```suggestion
> It is under **maintenance** and it won't have any major releases.
```

let's remove that line. I'm not sure when will it be deprecated. and new blade is already pushing out stable releases in chunks